src/Core/SokuAddresses.hpp

```diff
@@ -28,6 +28,7 @@ namespace SokuLib
 	#ifndef SOKU_VER_110
 		ADDR_RENDERER_BEGIN                         = 0x00401000,
 		ADDR_RENDERER_END                           = 0x00401040,
+		ADDR_LOAD_SE                                = 0x00401af0,
```

Comment:
You added this but didn't add a function anywhere that uses it. Did you forget it ?
Or didn't I undestand what this is properly ?

Reply:
i don't know exactly where to add it.

it is the loader for SFX, i used this to add more sounds.
since it is loaded very early in the game, they may be loaded before the module.

thiscall from 0x0089f9f8 with `void(int* addrToStoreId, char* filename)` signature
i used `0x00899D60 + id*4` as `addrToStoreId`, its the global sound table in the game

Reply:
Maybe you could put this in the SoundManager file ?
You could also add 0x00899D60 in the address list, make a new struct Sound and put it as an extern Sound ** in the SoundManager.hpp ?
Also 0x0089f9f8 is ADDR_TEXTURE_MANAGER (Which apparently is more of a resources manager) so you could add the function inside that.